extensions/wikia/AdEngine/js/lookup/services.js

```diff
@@ -25,22 +25,61 @@ define('ext.wikia.adEngine.lookup.services', [
 			rubiconFastlane,
 			prebid,
 			rubiconVulcan
-		];
+		],
+		bidIndex = {
+			'rubicon_fastlane': {
+				'pos': 0,
+				'char': 'R'
+			},
+			'ox_bidder': {
+				'pos': 1,
+				'char': 'O'
+			},
+			'amazon': {
+				'pos': 2,
+				'char': 'A'
+			},
+			'rubicon_vulcan': {
+				'pos': 3,
+				'char': 'V'
+			},
+			'prebid': {
+				'pos': 4,
+				'char': 'P'
+			}
+		},
+		bidMarker = ['x','x','x','x','x'];
```

Comment:
it doesn't have to be an array, it can be string.
You can access string's character using 'foo'[index]